src/utils/logger.js

```diff
@@ -0,0 +1,15 @@
+const logger = require('pino');
+
+module.exports = logger(
+  {
+    transport: {
+      target: 'pino-pretty',
+      options: {
+        colorize: true,
+        translateTime: 'SYS:dd-mm-yyyy HH:MM:ss',
+        ignore: 'pid,hostname',
+      },
+    },
+  },
+  logger.destination('../voluntera.log')
+);
```

Comment:
Have you tested that the logs are getting written to this log file?

Reply:
i tried to test it but no logs appear in the file, i think its only logs errors and higher level, but im not sure about it, the syntax of the code doesn't show any errors